packages/mantine/src/templates/FieldHelpTemplate.tsx

```diff
@@ -10,14 +11,21 @@ export default function FieldHelpTemplate<
   S extends StrictRJSFSchema = RJSFSchema,
   F extends FormContextType = any,
 >(props: FieldHelpProps<T, S, F>) {
-  const { fieldPathId, help } = props;
+  const { fieldPathId, help, uiSchema = {}, registry } = props;
 
   if (!help) {
     return null;
   }
 
   const id = helpId(fieldPathId);
-
+  const uiOptions = getUiOptions<T, S, F>(uiSchema, registry?.globalUiOptions);
```

Comment:
Make this patterned after the `FieldDescriptionTemplate` in this theme using the `RichHelp` component mentioned in the comment in `core`